src/components/membership-queue.ts

```diff
@@ -131,7 +131,7 @@ export class MembershipQueue {
         this.failureReasonCounter = metrics.addCounter({
             name: "membershipqueue_reason",
             help: "Count of failures to process membership, by matrix errcode and http status",
```

Comment:
This help lists all labels and should be updated (or have the list removed).

Reply:
Added type.